docroot/modules/custom/mass_admin_pages/mass_admin_pages.services.yml

```diff
@@ -4,3 +4,8 @@ services:
     tags:
       - {name: event_subscriber}
     arguments: ['@datetime.time', '@router.admin_context']
+
+  theme.negotiator.mass_admin_pages:
+    class: Drupal\mass_admin_pages\Theme\ThemeNegotiator
+    tags:
+      - { name: theme_negotiator, priority: -50 }
```

Comment:
Probably need a blank line at the end of this file.

Reply:
@weekbeforenext  sorry, I missed this comment before the merge.